'npm run build' fails on Windows systems

On my Linux system, I can run `npm run build` without problems.

On my Windows 10 box, the same command fails, however:

```
F:\DocsyFinal\docsy-ad>npm run build

> docsy@0.2.0 build
> npm run cd:docs build


> docsy@0.2.0 cd:docs
> npm run _cd:docs -- npm run "build"


> docsy@0.2.0 _cd:docs
> cd userguide && "npm" "run" "build"


> build
> npm run _build


> _build
> npm run _hugo-dev


> _hugo-dev
> npm run _hugo -- -e dev -DFE


> pre_hugo
> set -x && npm run ${DOCSY_VENDOR:-keep}:vendor

Die Umgebungsvariable "-x " ist nicht definiert.
npm ERR! code 1
npm ERR! path F:\DocsyFinal\docsy-ad\userguide
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c set -x && npm run ${DOCSY_VENDOR:-keep}:vendor

```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "scripts": {
     "_cd:docs": "cd userguide &&",
     "_check:format": "npx prettier --check .??* *.md",
-    "_cp:bs-rfs": "cp -R node_modules/bootstrap/scss/vendor/* assets/_vendor/bootstrap/scss/",
+    "_cp:bs-rfs": "npx cpy 'node_modules/bootstrap/scss/vendor/*' assets/_vendor/bootstrap/scss/",
     "_mkdir:hugo-mod": "npx mkdirp ../github.com/FortAwesome/Font-Awesome ../github.com/twbs/bootstrap",
     "_prebuild": "npm run _cp:bs-rfs",
     "_test:docs": "npm run cd:docs test",
@@ -26,7 +26,6 @@
     "postinstall": "npm run _mkdir:hugo-mod",
     "prebuild:preview": "npm run _prebuild",
     "prebuild:production": "npm run _prebuild",
-    "prepare": "npm run _cp:bs-rfs",
     "preserve": "npm run _prebuild",
     "pretest": "npm run _prebuild",
     "serve": "npm run cd:docs serve",
@@ -41,6 +40,7 @@
     "bootstrap": "5.2.3"
   },
   "devDependencies": {
+    "cpy-cli": "^5.0.0",
     "hugo-extended": "0.122.0",
     "markdown-link-check": "^3.11.2",
     "mkdirp": "^3.0.1",

